addon/components/sl-alert.js

```diff
@@ -87,12 +98,20 @@ export default Ember.Component.extend( TooltipEnabled, {
     /**
      * The generated Bootstrap "theme" style class for the alert
      *
-     * @function themeClassName
+     * @function
      * @observes theme
```

Comment:
@observes can be removed due to the intended use of https://github.com/notmessenger/jsdoc-plugins#emberobservestag